core/src/main/java/com/sap/ai/sdk/core/model/InlineObject.java

```diff
@@ -24,9 +24,9 @@
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
-/** FileDownload400Response */
+/** InlineObject */
 // CHECKSTYLE:OFF
-public class FileDownload400Response
```

Comment:
Not sure why this changed, but not worth investigating as these objects are unused anyway

Reply:
Weird :/ 

> these objects are unused anyway

True.